ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -1302,6 +1303,9 @@ Rectangle {
                             Binding on enabled {
                                 value: control.enabled
                             }
+                            Keys.onShortcutOverride: function (event) {
+                                event.accepted = event.matches(StandardKey.Paste)
```

Comment:
Handle the `Ctrl+V` shortcut separately, in order not to collide with the builtin TextEdit shortcuts